### DISCUSSION TITLE
Musicians profiles

### DISCUSSION THREAD

**Discussion by laurentlesax** [jamulussoftware/jamulus#2769] [Ideas]
Hello everyone.
It could be nice to have for each musician a more extended profile (_with picture, bio, email, homepage link, etc_) , so everyone can keep a track or contact a musician after a jam or keep in touch, or simply know who's playing...
Any plans about this in the future?

**Comment by chrisrimple** [comment]
Keep in mind that Jamulus stores nothing about the users on any server, so everything that appears in a User Profile must be stored locally on the musician's computer and will only be visible to other musicians on the same server at the same time.  That said, it probably wouldn't be hard to add a couple new edit fields for EMAIL, URL, and BIO, and store those in the INI file.  Photo is a little more complex, since a path to a JPG (or similar) would need to be stored, then that JPG needs to be rendered in the UI somewhere (probably on mouseover, but maybe replace the country flags with a shrunken version of the JPG).

**Reply by laurentlesax** [reply]
Yes. Keeping Jamulus simple and stable is the main thing. But as you said, maybe just add a couple of extra fields in the profile maybe (?) would be a + ;-)

**Comment by pljones** [comment]
If you want to exchange your Facebook details with someone, or the URL of an MPEG of you playing in a concert, just post them chat.

Jamulus is for jamming.  We're keeping it simple.  There's already a lot of bloat causing problems.

**Reply by laurentlesax** [reply]
i understand that. Didn't think of something too complex. Just extend the profile fields in the parameters... ;-)

**Comment by ann0see** [comment]
@laurentlesax Thanks for your suggestion. We had the request of social features a lot in the past, but usually decided against it for various reasons (complexity, privacy, current design,…). However, since the community is an integral part of Jamulus, there are services (and also just the chat, for sure). 
I think there’s a service which adds social features: jamulus.live and Jamulus lounge. Probably there are many more. 

If you’re curious on why social features etc are not included, you can read the paragraph about KISS in the following file: https://github.com/jamulussoftware/jamulus/blob/master/CONTRIBUTING.md

**Reply by laurentlesax** [reply]
Thank you for your clear reply. I understand completely. It was just a suggestion... ;-)
Jamulus.live could be improved then maybe (?) ... with more features than just a list of connected people.
Whatever, i will still enjoy very much Jamulus, for sure!

**Reply by ann0see** [reply]
Yeah, sure. I don't know who's behind jamulus.live, but maybe the site has a contact developer button?

**Reply by laurentlesax** [reply]
Didn't find any contact button on jamulus.live :-(

**Reply by pljones** [reply]
There's also places like [worldjam.vip](https://worldjam.vip) (see their [YouTube channel](https://www.youtube.com/c/WorldJam), too) - i.e. sites that specialise in arranging events over Jamulus - where the social side is expanded well outside Jamulus: they've a [Facebook group](https://www.facebook.com/groups/JamulusWorldJam) and hold meet ups every now and then (there have been a couple in the UK, one coming up in the US, I think and one next year in The Netherlands).  They just use Jamulus for what it's for.  (You don't expect your mouse to have social features or your drum kit or ...)

That's just one example.  I mentioned Facebook earlier, as I know there are other places there, too.
